### PR TITLE
faxsms fixes

### LINKED ISSUE #7817: Fix various bugs in faxsms module.
### Describe the bug
Secure messaging Send SMS broken.
Rest API sendSMS() args conflicts.
Testing SMS logs from live system to fine tune client.
### Client configuration

- Browser:
- OpenEMR version: v7.0.2.2
- Operating system:

### PR DESCRIPTION
**Pull request by sjpadgett**
RC setup from services menu wasn't refreshing after save. 
ensure phone numbers are formatted correctly throughout.
fix RC sendSMS for rest API endpoint.

Fixes #7817

### CODE REVIEW

**Comment by sjpadgett** [comment]
image

File: interface/modules/custom_modules/oe-module-faxsms/src/Controller/RCFaxClient.php [hunk 1]
```diff
@@ -198,23 +198,28 @@ public function faxProcessUploads(): string
...
+        $toPhone = $this->formatPhone($toPhone);
         if ($smsNumber) {
             try {
                 $this->platform->post('/account/~/extension/~/sms', [
                     'from' => ['phoneNumber' => $smsNumber],
                     'to' => [['phoneNumber' => $toPhone]],
                     'text' => $message,
                 ]);
-                sleep(1);
+                sleep(1.25);
```
Comment (stephenwaite) [inline interface/modules/custom_modules/oe-module-faxsms/src/Controller/RCFaxClient.php:222]: signal-2024-02-26-13-04-15-395

**Comment by sjpadgett** [comment]
@bradymiller I don't like the menu test! my commit has nothing to do with finder

**Comment by bradymiller** [comment]
@sjpadgett , there was a time when we were getting about 10% false positive in our e2e tests that was only doing about 7 tests per run. Now i think getting about 2-5% false positive in our e2e tests that is now doing about 150+ tests per run. At some point plan to see what is going on with that undefined error that occasionally pops up in order to further decrease the false positive (it has been pretty painful and have already worked in bunch of mechanisms to decrease the false positive tests). I don't even know how I got sucked into the e2e stuff to begin with :)

**Comment by bradymiller** [comment]
2DV

**Comment by sjpadgett** [comment]
Sorry @bradymiller I didn't mean to offend and we appreciate your efforts. I have never put a lot of faith in automated functional testing but I'm a dinosaur and certainly not up to date on their usefulness. 
You know me, if my mouth is open, my foot isn't far behind.:)

**Pull request merged**